apiserver/facades/client/spaces/spaces_rename.go

```diff
@@ -0,0 +1,255 @@
+/*
+ * // Copyright 2020 Canonical Ltd.
+ * // Licensed under the AGPLv3, see LICENCE file for details.
+ */
+
+package spaces
+
+import (
+	"github.com/juju/errors"
+	"github.com/juju/juju/apiserver/common/networkingcommon"
+	"gopkg.in/juju/names.v3"
+	"gopkg.in/mgo.v2/txn"
+
+	"github.com/juju/juju/apiserver/common"
+	"github.com/juju/juju/apiserver/params"
+	jujucontroller "github.com/juju/juju/controller"
+	"github.com/juju/juju/core/constraints"
+	"github.com/juju/juju/core/permission"
+	"github.com/juju/juju/core/settings"
+	"github.com/juju/juju/state"
+)
+
+// RenameSpaceModelOp describes a model operation for committing a branch.
+type RenameSpaceModelOp interface {
```

Comment:
I don't think you need this interface if it doesn't extend `ModelOperation`.

Reply:
I removed the interface and just used the state.ModelOperation in the signature for the other function using this. As I am not needing the interface indirection I think.